packages/components-js/tests/e2e/puppeteer/specs/link-tile-product.e2e.ts

```diff
@@ -1,6 +1,6 @@
 import { expectA11yToMatchSnapshot, getHTMLAttributes, selectNode, setContentWithDesignSystem } from '../helpers';
 import type { Page } from 'puppeteer';
-import { Components } from '@porsche-design-system/components';
+import type { Components } from '@porsche-design-system/components/src/components';
```

Comment:
Why do we need a `src` cross import instead of a package import?

Reply:
This is needed so that the tests can be run without building the packages